assets/training/model_management/src/azureml/model/mgmt/processors/pyfunc/vision/mmdet-od-requirements.txt

```diff
@@ -5,4 +5,5 @@ openmim==0.3.9
 torch==2.0.1
 torchvision==0.15.2
 transformers==4.34.0
-accelerate==0.23.0
\ No newline at end of file
+accelerate==0.23.0
+fairscale==0.4.13
```

Comment:
would like to know the exact purpose of using both packages here!

Reply:
fairscale is used by mmd and accelerate is used by hf trainer